pkg/auth/auth.go

```diff
@@ -279,15 +271,44 @@ func LoginWithPKCE(host, publicClientID string, tlsNoVerify bool) (*oauth2.Token
 
 // Revokes the access token
 func RevokeAccessToken(endpoint, publicClientID, refreshToken string, tlsNoVerify bool) error {
-	pCfg, err := getPlatformConfiguration(endpoint, publicClientID, tlsNoVerify)
+	rp, err := newOidcRelyingParty(context.Background(), endpoint, tlsNoVerify, oidcClientCredentials{
+		clientID: publicClientID,
+		isPublic: true,
+	})
 	if err != nil {
-		return fmt.Errorf("failed to get platform configuration: %w", err)
+		return err
+	}
+	return oidcrp.RevokeToken(context.Background(), rp, refreshToken, "refresh_token")
```

Comment:
Note for later: we should take the existing command context in all these functions I think.

Reply:
Yes I agree.